hathor/wallet/resources/send_tokens.py

```diff
@@ -309,7 +309,7 @@ def render_OPTIONS(self, request):
                                             'inputs': [],
                                             'outputs': [],
                                             'tokens': [],
-                                            'accumulated_weight': 14
+                                            'accumulated_weight': '16384'
```

Comment:
I guess we won't do it anymore, right? I mean, it will return `accumulated_weight: 14` and `accumulated_weight_raw: '16384'`, right?

Reply:
Fixed.

Reply:
Thanks!